### DISCUSSION TITLE
Current state of packages / Funding

### DISCUSSION THREAD

**Discussion by dnish** [Meteor-Community-Packages/organization#76] [General]
Hey guys,
thank you for the amazing work. We are currently trying to upgrade our MeteorJS skeleton to version 3. We've noticed that some of the packages are currently not working anymore:

redis-oplog
fast-render

Are there any plans to update them for Meteor 3 and is someone currently working on them? I've tried to find the funding section which was mentioned somewhere in the CONTRIBUTING.md but couldn't find anything here.

**Comment by StorytellerCZ** [comment]
They are on the list, but no one is actively working on them as far as I know from the people in the organization. But there are [people working on redis-oplog](https://github.com/Meteor-Community-Packages/redis-oplog/pull/13).

As for funding, each repository there should be `Sponsor this project` section that lists the maintainers. That is most often me and @jankapunkt

**Reply by dnish** [reply]
Hey, thanks for the reply. For redis-oplog there is a section at the bottom, but it redirects to the Open Collective account of the creator, so I'm not sure if he is still active on the project? I didn't notice the sponsor this project section on the right because I was looking for some information within the README.md / CONTRIBUTING.md 😀.

For fast-render I see the link to copleky who submitted some code about 3 years ago.

**Reply by StorytellerCZ** [reply]
On most projects you will see something like this:
![image](https://github.com/Meteor-Community-Packages/organization/assets/1715235/ea00139d-8d7a-410c-a7fe-75077385fa11)
